spec/features/obituaries/create_spec.rb

```diff
@@ -100,6 +118,6 @@
   it "I cannot access the obituary create form" do
     obituary = create(:obituary)
     visit new_obituary_path(obituary.id)
-    expect(page).to have_content('The page you were looking for doesn\'t exist (404)')
+    expect(page).to have_content("The page you are looking for does not exist")
```

Comment:
All I think of when I hear the phrase "does not exist"...

![](https://media0.giphy.com/media/7JvlHfd7C2GDr7zfZF/giphy.gif?cid=5a38a5a2ol569uggh2abbx3rp27av9cmoedeg4plk78q1uf6&rid=giphy.gif)